src/script/api/script_viewport.hpp

```diff
@@ -28,6 +30,38 @@ class ScriptViewport : public ScriptObject {
 	 * @pre ScriptMap::IsValidTile(tile).
 	 */
 	static void ScrollTo(TileIndex tile);
```

Comment:
Should ScrollTo be kept?
Or should it be removed and get mapped to ScrollEveryoneTo via the compatibility layer?

Reply:
Dunno. I think of new scrolling methods more like of a mp-only thing, even though some of them work in sp and, I guess, end result will be the same as with ScrollTo. Though now that I added ResetObjectToPlace it's a bit different, so we can keep both just for the sake of it.

Reply:
In other words old ScrollTo is a pure interface method, but new ones are more gameplay-related. Even though they are implemented mostly the same atm they are logically different and could evolve in different directions in future.